dependency_support/pip_requirements.in

```diff
@@ -13,6 +13,7 @@ pyyaml==6.0.1
 # We build most of z3 ourselves but building python is really complicated. Just
 # use pypi version
 z3-solver==4.14.0.0
+zstandard==0.23.0
```

Comment:
as discussed is it possible to target `0.19.0` for now?

Reply:
Sure, I will check that